### PR TITLE
Feature/140 mark unused code

### PR DESCRIPTION
**Pull request by st161630**

### CODE REVIEW

File: tests/automated_tests/conftest.py
Comment (BonnetGa) [inline tests/automated_tests/conftest.py:1]: Why do we need this changes ?

File: tests/job_test_data.json
Comment (BonnetGa) [inline tests/job_test_data.json:1]: Why do we need this ?

File: tests/automated_tests/conftest.py
Comment (BonnetGa) [reply tests/automated_tests/conftest.py:1]: Maybe you should also mark the tests for this experimental features and remove it from the automated tests ?

Comment (st161630) [reply tests/automated_tests/conftest.py:1]: In fact the tests are not only unncessary but I have just realised that even the functionality currently present in the lines of code is no longer compatible with the changes we have made around it. I will comment the tests out, however doing the same for the methods, or possibly not even allowing them might be the right call.

**Review by st162107 — APPROVED** [review]

**Pull request merged**